src/Api/Serializer/JsonRpcSerializer.php

```diff
@@ -68,15 +67,8 @@ public function __invoke(
                 'Content-Type' => $this->contentType
             ];
 
-        if ($endpointProvider instanceof EndpointProviderV2) {
-            $this->setRequestOptions(
-                $endpointProvider,
-                $command,
-                $operation,
-                $commandArgs,
-                $clientArgs,
```

Comment:
We're just changing the entrypoint from the serializer to the new `EndpointV2Middleware`.  We don't need them in the serializer now since we only need to expect a resolved endpoint object.

Reply:
Understood!